draft-ietf-quic-transport.md

```diff
@@ -3022,85 +3026,107 @@ processed as valid.
 The server MUST also limit the number of bytes it sends before validating the
 address of the client, see {{address-validation}}.
 
+## Path Maximum Transmission Unit (PMTU)
 
-## Path Maximum Transmission Unit
+The PMTU is the maximum size of the entire IP datagram including the IP header,
+UDP header, and UDP payload.  The UDP payload includes the QUIC packet header,
+protected payload, and any authentication fields. This can be depend upon the
+current path characteristics.  Therefore, the current largest UDP payload an
+implementation will send is referred to as QUIC Maximum Packet Size (MPS).
```

Comment:
We use term MPS in DPLPMTUD, we have found that keeping the various packet size
values straight is quite hard. It is easier to reason about the sizes when
you seperate away the path size (PMTU), the estimate of the path size and the
size the transport can send (MPS).

I would not object to using 'maximum packet size' rather than an acronym.